Sites/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>GD2 PR2</title>
+    <link rel="stylesheet" href="Styles.css">
+</head>
+
+<body>
+    <div>
+        <div class="container clearfix">
+            <h1>
+                <div class="underline">A Day in Chicago</div>
+            </h1> <br>
+
+            <div class="laptop">
+
+                <h2>Follow me through a day over my spring break while visiting my sister at The School of the Art
+                    Institute of Chicago! Impressively, I stayed awake from 10 AM to 11 PM, scroll to view my day in
+                    hourly
+                    photos! </h2>
+            </div>
+
+            <div class="laptop">
+
+                <h3>
```

Comment:
I think you could make your layout less brittle by updating your markup and using `display: inline-block` vs the `position: relative;` styles you've used to make the captions appear to the right of the photo. For example:

```html
<figure>
  <img src="img/Hour-1.jpg" alt="Picture looking up at the above-ground train and skyscrapers">
    <figcaption>
        <h3>Hour 1 [ 10:16 ]</h3>
        <p>This photo is from a sunny Chicago morning on Thursday, March 1st. On the way to breakfast, we stopped to admire the city's amazing arcitecture.</p>
    </figcaption>
</figure>
```
Then tweak your CSS to control the width of the `img` and make the `img` and `ficaption` elements both `display:inline-block;`